planner/core/plan_stats_test.go

```diff
@@ -43,7 +43,7 @@ func TestPlanStatsLoad(t *testing.T) {
 	tk.MustExec("drop table if exists t")
 	tk.MustExec("set @@session.tidb_analyze_version=2")
 	tk.MustExec("set @@session.tidb_partition_prune_mode = 'static'")
-	tk.MustExec("set @@session.tidb_stats_load_sync_wait =9999999")
```

Comment:
Why do we modify the value? Does that mean the loading time is usually less than 1 min in the test cases so `60000` is enough.

Reply:
Yes, if there's anything wrong, waiting for 60s is enough to fail this sql.